priv/repo/migrations/20170509093107_add_likes_table.exs

```diff
@@ -0,0 +1,11 @@
+defmodule App.Repo.Migrations.AddLikesTable do
+  use Ecto.Migration
+
+  def change do
+    create table :likes do
+      add :user_hash, :string
+      add :article_id, :integer
+      add :like_value, :integer
```

Comment:
Just curious why you choose to represent the likes with integer? My first thought is to use Boolean: true -> like; false -> dislike; nil -> no value